Sources/Lifecycle/Lifecycle.swift

```diff
@@ -197,6 +197,18 @@ public struct ServiceLifecycle {
     /// - parameters:
     ///    - configuration: Defines lifecycle `Configuration`
     public init(configuration: Configuration = .init()) {
```

Comment:
I think, we should update the header comment. It should say something like, there should only be one ServiceLifecycle per executable. For composition always use `ComponentLifecycle`. Creating a second `ServiceLifecycle` will lead to a crash.